src/components/List.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { updatePurchaseDate } from '../lib/firebase.js';
 
-const List = ({ token, results }) => {
+const List = ({ results, token }) => {
+  function handleOnCheck(event) {
+    updatePurchaseDate(token, event.target.value);
+  }
```

Comment:
I saw the AC requires us to disable shopping items for 24 hours but I'm wondering if it might be a useful ability to fix mistakenly checking a box. 

I think it would be possible if in the `handleCheck` function there was a condition check to see if the item is checked (`event.target.checked`). If it is not checked then run a separate update function that uses the `arrayRemove` firebase function and passes in the last purchase date which is currently saved as the `time` variable. Otherwise, run the current `updatePurchaseDate` function.

Might not be necessary, but I think it would be a welcome addition!

Reply:
Great call! I think that we could add an "undo last purchase" next to the "last purchase date" on the item details screen quite easily without allowing users to edit their most recent purchase date and also meet the AC which would not allow the item to be flagged as purchased twice within 24 hours. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/item-details.png

Reply:
That's sounds like a great way to deal with both the AC and added functionality.

Reply:
Added to issue #13 !